W-14745160-redis-docs-update-kt (#2597)
mulesoft/docs-connectors: redis/5.4/modules/ROOT/pages/redis-connector-reference.adoc

[NOTE]
You can set configurations in Expression mode, except when Redis Connector is used in conjunction with Object Store Connector, which does not support Expression mode.

> Starting with Redis Connector 5.4.5, Mule apps that use an Object Store *Entry TTL* exceeding 24 days, now observe that the TTL for new keys is set to the correct values.

Reviewer: ```suggestion
Starting with Redis Connector 5.4.5, Mule apps that use an Object Store *Entry TTL* exceeding 24 days now observe that the TTL for new keys is set to the correct values.
```

> Starting with Redis Connector 5.4.5 the *Entry TTL* set on the object store for periods exceeding 24 days now have the correct value for the new object store keys created.

Reviewer: ```suggestion
Starting with Redis Connector 5.4.5, *Entry TTL* now has the correct value for the new object store keys created for periods exceeding 24 days.
```

Maybe sth like this?
